m0t1fs/linux_kernel/st/m0t1fs_server_inc.sh

```diff
@@ -95,7 +95,7 @@ EOF
 	for (( ; DDEV_ID < $dev_end; DDEV_ID++)) ; do
 		conf_ios_device_setup $DDEV_ID $id_count id_count "$ids" ids
 
-		dd if=/dev/zero of=$DDEV_ID$ddisk bs=1M seek=1M count=1 ||
+		dd if=/dev/zero of=$DDEV_ID$ddisk bs=1M seek=$IOS_DISK_BLOCK_COUNT count=1 ||
```

Comment:
Rename `IOS_DISK_BLOCK_COUNT` to `IOS_DISK_SEEK_OFFSET`?  It's definitely not block count, that's `count=1` option.

Reply:
Ok, better suggestion: `IOS_DISK_SEEK_BLOCK_COUNT`.  Option `seek` will skip specified number of blocks at the beginning of output file.

Reply:
Because we use "seek", so the final file has ($IOS_DISK_BLOCK_COUNT + 1) blocks: the first $IOS_DISK_BLOCK_COUNT blocks are sparse, and only the last is actually created.

That is approximately equal to $IOS_DISK_BLOCK_COUNT blocks in total.
Maybe we can have:
dd if=/dev/zero of=$DDEV_ID$ddisk bs=1M seek=$(IOS_DISK_BLOCK_COUNT - 1) count=1

Reply:
OK. Fixed.